Apeswap-Bots/detect-large-banana-token-mints/src/agent.ts

```diff
@@ -0,0 +1,62 @@
+import { Finding, HandleTransaction, TransactionEvent, getEthersProvider } from "forta-agent";
+import { providers, BigNumber } from "ethers";
+import { formatEther } from '@ethersproject/units'
+import NetworkManager, { NETWORK_MAP } from "./network";
+import NetworkData from "./network";
+import TotalSupplyFetcher from "./total.supply.fetcher";
+import { createFinding, providerParams, providerParamsType, threshold } from "./utils";
+
+
+const networkManager = new NetworkManager(NETWORK_MAP);
+const ethersProvider: providers.Provider = getEthersProvider();
+const totalSupplyFetcher: TotalSupplyFetcher = new TotalSupplyFetcher(ethersProvider, networkManager);
+
+export const initialize = (provider: providers.Provider) => {
+  return async () => {
+    const { chainId } = await provider.getNetwork();
+    networkManager.setNetwork(chainId);
+    totalSupplyFetcher.setBananaContract();
+  };
+};
+
+export let exportedNetwork: string;
```

Comment:
This is not used(?)

Reply:
that was absolutely an oversight, removed :)